examples/real-world/routes.js

```diff
@@ -1,14 +1,16 @@
 import React from 'react'
-import { Route } from 'react-router'
+import { Route, browserHistory, Router } from 'react-router'
 import App from './containers/App'
 import UserPage from './containers/UserPage'
 import RepoPage from './containers/RepoPage'
 
-export default (
-  <Route path="/" component={App}>
-    <Route path="/:login/:name"
-           component={RepoPage} />
-    <Route path="/:login"
-           component={UserPage} />
-  </Route>
-)
+export default function Routes() {
+  return (<Router history={browserHistory}>
+            <Route path="/" component={App}>
+              <Route path="/:login/:name"
+                     component={RepoPage} />
+              <Route path="/:login"
+                     component={UserPage} />
+            </Route>
+          </Router>)
```

Comment:
Small nitpick here: Would you might keeping the original indentation? The less width in your code, the better. It's normal to do something like this with JSX:
```js
return (
   <SomeComponent />
)
```